custom/icds/management/commands/contact_phone_fix.py

```diff
@@ -0,0 +1,178 @@
+import csv
+import six
+import sys
+import time
+
+from datetime import (
+    datetime,
+    date,
+    timedelta,
+)
+from xml.etree import cElementTree as ElementTree
+
+from django.core.management.base import BaseCommand
+
+from corehq.apps.users.util import SYSTEM_USER_ID
+from corehq.form_processor.backends.sql.dbaccessors import CaseReindexAccessor
+from corehq.form_processor.interfaces.dbaccessors import CaseAccessors
+from corehq.util.log import with_progress_bar
+from corehq.apps.locations.models import SQLLocation
+from corehq.apps.hqcase.utils import submit_case_blocks
+from corehq.form_processor.backends.sql.dbaccessors import iter_all_rows
+
+from dimagi.utils.chunked import chunked
+from casexml.apps.case.mock import CaseBlock
+
+DOMAIN = "icds-cas"
+CASE_TYPE = "person"
+
+DATE_OF_REGISTRATION_PROPERTY = "date_of_registration"
```

Comment:
Is this property used somewhere?
If not, can we remove it?

Reply:
removed!